docs/contributing/index.md

```diff
@@ -156,6 +158,24 @@ To standarize the developer experience across packages, a configuration tool is
 
 See the [tool documentation](https://github.com/plone/meta) for more information.
 
+
+(contributing-guidance-for-merging-pull-requests-label)=
+
+## Guidance for merging pull requests
+
+-   If you have any doubt, don't merge, but ask.
+-   All {doc}`CI checks <core/continuous-integration>` must pass, unless there is a good reason to ignore it.
+    If you ignore a check, then write the good reason either in a comment or in the merge message.
+-   Get at least one approval from another Plone contributor, preferably a member of the GitHub Team for the repository.
+-   If the pull request has documentation, ask for a review from `@plone/documentation-team`.
+-   You don't need to write anything in the merge message, as it usually populates with all the commit messages.
```

Comment:
I did a merge in Volto with squash and it didn't include the commit messages in the description. So it depends on the repository.